TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml

```diff
@@ -2,9 +2,7 @@
 @model TheModernIlluminati.Pages.LaureateModel
 @{
     ViewData["Title"] = "Laureate";
-    List<TheModernIlluminati.Models.Laureate>
-    laureate1 = (List<TheModernIlluminati.Models.Laureate>
-        )ViewData["Laureates"];
+    List<TheModernIlluminati.Models.Laureate> laureates = (List<TheModernIlluminati.Models.Laureate>)ViewData["Laureates"];
```

Comment:
Replaced laureate1 with laureates.

sources: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Reply:
Yeah, the 1 isn't adding much value to the variable name.